frontend/admin/src/js/components/apiManagedTable/BasicTable.tsx

```diff
@@ -37,18 +37,27 @@ function BasicTable<T extends RecordWithId>({
     }
     // no current sorting, sort by this column
     if (!sortingRule) {
-      onSortingRuleChange({ column, desc: false });
+      onSortingRuleChange({
+        column: { id: column.id, sortColumnName: column.sortColumnName },
+        desc: false,
+      });
```

Comment:
Is this to avoid Typescript trouble?

Reply:
No, the column value passed was changed because the `SortingRule` type needed to change its shape in order to be able to sort by a column initially (by default):

https://github.com/GCTC-NTGC/gc-digital-talent/pull/5177/files#diff-659802ca88b564f0c8f0476bd7286693f098c53e6c8e4fcf8cac43e1cbdd5465